content/blog/2017-11-30-prometheus-at-cloudnativecon.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Prometheus at CloudNativeCon 2017
+created_at: 2017-11-30
+kind: article
+author_name: Tom Wilkie on behalf of the Prometheus team
+---
+
+## Prometheus at CloudNativeCon 2017
+
+Wednesday 6th December is Prometheus Day at CloudNativeCon Austin, and we’ve got
+a fantastic lineup of talks and events for you.  Go to the Prometheus Salon for
+hands on advice on how best to monitor Kubernetes, attend a series of talks on
+various aspects of Prometheus and then hang out with some of the Prometheus developers at the CNCF
+booth, all followed by the Prometheus Happy Hour.  Read on for more details...
+
+<!-- more -->
+
+##  [Prometheus Salon](https://kccncna17.sched.com/event/Cs4d/prometheus-salon-hosted-by-frederic-branczyk-coreos-bob-cotton-freshtracksio-goutham-veeramanchaneni-tom-wilkie-kausal) (11:10am - 12:30pm)
+
+Featuring talks from Prometheus developers, this salon will include an introduction
+to Prometheus, a closer look at how you can use Prometheus to monitor your Kubernetes
+cluster, and a discussion of the new features in Prometheus 2.0.  The session will
+include hands-on access to a live Prometheus and Kubernetes cluster, allowing you
+to experiment with PromQL queries to gain deeper insights into your Kubernetes clusters.
+
+## [The RED Method: How To Instrument Your Services](https://kccncna17.sched.com/event/CU8K/the-red-method-how-to-instrument-your-services-b-tom-wilkie-kausal) (2:45pm - 3:20pm)
+
+Tom from Kausal will cover patterns of application instrumentation, where and when they are applicable, and how they can be implemented with Prometheus.  Covering Google’s Four Golden Signals, the RED Method and the USE Method, this talk will also discuss why consistency is an important approach for reducing cognitive load. Finally it will discuss the limitations of these approaches and what can be done to overcome them.
+
+## Meet the Maintainers (3:30pm)
+
+Prometheus developers Frederic Branczyk, Goutham Veeramachaneni and Tom Wilkie will
```

Comment:
Most people know us by either our twitter or github profile, maybe worth linking out to.